app/helpers/wizard/event.py

```diff
@@ -414,9 +414,13 @@ def save_resized_background(background_image_file, event_id, size, image_sizes):
 def save_social_links(social_links, event):
     for social_link in social_links:
         if social_link['link'].strip() != "":
+            if not social_link['link'].startswith("http"):
+                social_link['link'] = "https://" + social_link['link']
+            else:
+                social_link['link'] = social_link['link']
             social_exists = SocialLink.query.filter_by(name=social_link['name'], event_id=event.id).scalar()            
             if social_exists:
                 SocialLink.query.filter_by(name=social_link['name'], event_id=event.id).update({'link': social_link['link']})
             else:
                 social = SocialLink(social_link['name'], social_link['link'], event.id)
-                db.session.add(social)
+                db.session.add(social)
```

Comment:
@magdalenesuo please don't remove the newline at the end. It is in the coding standard.